docs/user/implement_algo_advanced.rst

```diff
@@ -92,7 +92,7 @@ extra diagnostic information as well as supporting recurrent policies):
             'action',
             batch_dims=1,
         )
-        advantage_var = tf.placeholder('advantage')
+        advantage_var = tf.compat.v1.placeholder('advantage')
```

Comment:
does it throw deprecation warnings if you don't use the compat.v1 modules?

Reply:
Yes

Reply:
alright